introduction.md

```diff
@@ -0,0 +1,2 @@
+This is the file that we are supposed to use for the introduction of ourselves to the dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.